src/test/resources/county-court-judgment/invalid-defendant-type.json

```diff
@@ -0,0 +1,5 @@
+{
+  "defendant": {
```

Comment:
not relevant to CCJ, this is related to party type

Reply:
removed